navbar.php

```diff
@@ -28,7 +28,7 @@
           echo "<li><a href='./index.php'>Logged in as ".$_SESSION['email']."</a></li>";
           echo "<li><a href='./logout.php'>LOGOUT</a></li>";
         } else {
-          echo "<li><a href='./signin.php'>LOGIN</a></li>";
+          echo "<li><a href='./signin.php?err='>LOGIN</a></li>";
```

Comment:
If err is not there in the url the php prompts with an error of undefined variable so i thought to keep it just to remove the error